facebook/__init__.py

```diff
@@ -237,6 +246,12 @@ def request(
             elif "access_token" not in args:
                 args["access_token"] = self.access_token
 
+        if self.app_secret_hmac:
+            if post_args is not None:
+                post_args["appsecret_proof"] = self.app_secret_hmac
+            else:
+                args["appsecret_proof"] = self.app_secret_hmac
+
         try:
             response = requests.request(method or "GET",
                                         FACEBOOK_GRAPH_URL + path,
```

Comment:
If this commit is exactly the same as the two commits in #243, could you [change the author of the commit](https://stackoverflow.com/a/3042512/113527) to @dimitrovskif?